src/components/registerCodeForm.tsx

```
@@ -1,95 +1,165 @@
-    title: {
-      fontSize: rem(26),
-      fontWeight: 900,
-      fontFamily: `Greycliff CF, ${theme.fontFamily}`,
-    },
+import { useAuthState } from 'react-firebase-hooks/auth';
+import { auth } from "@/service/UI/firebaseUiClient";
+import { css } from '@emotion/react';
+import Link from 'next/link';
+
+type Props = {
+  confirm: Function
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)